### PR TITLE
[fix] 100% width on iPhone 5 portrait

### PR DESCRIPTION
**Pull request by khanghoang**
WIP #515 

iphone 6 ios 10 3 14e8301 2017-10-29 16-02-41
iphone 5 ios 10 3 14e8301 2017-10-29 15-57-41

### CODE REVIEW

**Comment by khanghoang** [comment]
@bluepnume is there any particular reason for this style that you can think of? https://github.com/paypal/paypal-checkout/blob/master/src/components/checkout/templates/container/style.js#L173
and this one https://github.com/paypal/paypal-checkout/blob/master/src/components/checkout/templates/container/style.js#L177

File: src/components/checkout/templates/container/style.js [hunk 1]
```diff
@@ -374,7 +374,9 @@ export function getContainerStyle({ id, tag, CONTEXT, CLASS, ANIMATION } : { id
         }
 
         #${ id }.${ tag }-context-${ CONTEXT.IFRAME } .${ CLASS.OUTLET } iframe {
-            width: 100%;
+            width: 1px;
```
Comment (bluepnume) [inline src/components/checkout/templates/container/style.js]: Not sure I understand the fix -- could you explain the 1px and the *width?

**Comment by bluepnume** [comment]
The styles you linked -- I don't think they're 100% necessary, so long as the text stays centered. (BTW no need to work on this on the weekend, it's not hugely urgent!)

File: src/components/checkout/templates/container/style.js (hunk 1 above)
Comment (khanghoang) [reply src/components/checkout/templates/container/style.js]: The close button is clipped on iphone 5 portrait mode
iphone 5 ios 10 3 14e8301 2017-10-29 17-05-26
more details here https://stackoverflow.com/questions/23083462/how-to-get-an-iframe-to-be-responsive-in-ios-safari?answertab=votes#tab-top

**Comment by khanghoang** [comment]
@bluepnume I do this just because just because I got nothing else to do. It's just for fun.

File: src/components/checkout/templates/container/style.js (hunk 1 above)
Comment (bluepnume) [reply src/components/checkout/templates/container/style.js]: Makes sense! I think we can drop the *width here though since we only need to support IE9+

**Comment by khanghoang** [comment]
@bluepnume is there any particular reason for these styles that you can think of? https://github.com/paypal/paypal-checkout/blob/master/src/components/checkout/templates/container/style.js#L173
and this one https://github.com/paypal/paypal-checkout/blob/master/src/components/checkout/templates/container/style.js#L177

**Comment by bluepnume** [comment]
I don't see any reason for them to be there, you can feel free to remove them so long as the text in that overlay stays centered.

**Comment by bluepnume** [comment]
Merged! Thanks @khanghoang

**Comment by khanghoang** [comment]
@bluepnume this is WIP PR, I wanted to confirm with you about my concerns about those styles before I make any change. 
This PR didn't fix the overflowed text, it only fixes the clipped close button. Therefore, I need to make another PR.

**Comment by bluepnume** [comment]
Ah sorry, didn't realize it was WIP

**Pull request merged**